StratoSnap/MainFlightController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  ViewController.swift
+//  StratoSnap
+//
+//  Created by Abhishek  Ravi on 2018-12-15.
+//  Copyright © 2018 Abhishek  Ravi. All rights reserved.
+//
+
+import UIKit
+import Photos
+import AVFoundation
+import Vision
+/// TODO: add http requests to drones control system indicating whether to position gimbal up or down based on adjustment algorithm
+class MainFlightController: UIViewController {
+    
+    //@IBOutlet var capturePreviewView: PreviewView!
+    
+    let cameraController = CameraController()
+    
+    
+    private lazy var capturePreviewView: PreviewView = {
+        let view = PreviewView()
+        view.backgroundColor = #colorLiteral(red: 0, green: 0, blue: 0, alpha: 1)
+        view.alpha = 1.0
+        return view
+    }()
+    
+    var info: FlightSettings?
+    
+    @objc func captureClicked(_ sender: Any) {
+        cameraController.captureImage {(image, error) in
+            guard let image = image else {
+                print(error ?? "Image capture error")
+                return
+            }
+            print("Image Captured!")
+            
+            try? PHPhotoLibrary.shared().performChangesAndWait {// save to library
+                PHAssetChangeRequest.creationRequestForAsset(from: image)
+            }
+        }
+    }
+    ///Displays a preview of the video output generated by the device's cameras.
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        AppUtility.lockOrientation(.portrait)
```

Comment:
so I made it portrait, it currently doesnt lock in ".landscape" when orientation switches, it switches too.